medusa/storage/s3_storage.py

```diff
@@ -12,29 +12,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import base64
 import sys
 import requests
 import configparser
 import logging
 import os
 import io
-import itertools
-import subprocess
-from subprocess import PIPE
-from dateutil import parser
-from pathlib import Path
 
 from libcloud.storage.providers import get_driver
 
-from medusa.storage.abstract_storage import AbstractStorage
-import medusa.storage.aws_s3_storage.concurrent
-from medusa.storage.aws_s3_storage.awscli import AwsCli
+from medusa.storage.s3_base_storage import S3BaseStorage
 
-import medusa
 
-
-class S3Storage(AbstractStorage):
+class S3Storage(S3BaseStorage):
     """
     Available storage providers for S3:
     S3_AP_NORTHEAST = 's3_ap_northeast'
```

Comment:
We should remove these in favor of using `s3` as storage provider and setting the region instead of using region specific storage providers.